<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="INCNU4129FFM" ignored="0" name="TestFour" tests="1" failures="1" timestamp="2020-11-16T14:54:34 IST" time="58.977" errors="0">
  <testcase name="TestFour" time="52.284" classname="test.FourTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.FourTest.TestFour(FourTest.java:22)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- TestFour -->
</testsuite> <!-- TestFour -->
